### PR TITLE
change hard-code of rowsCount

### PR DESCRIPTION
**Pull request by Benjamin-GitHub**
Now we have rows with no specific desk as much as we have booking per day.

### CODE REVIEW

**Review by viktoriia-kosenko — DISMISSED** [review]
Hey Benjamin, just checked your branch, it looks like after merging Rebeka's changes your branch has errors. Could you please pull the branch and run it locally. let me know if you need help fixing errors

**Comment by gregdyke** [comment]
I could be mistaken, but looking at the files changed, I don't see any errors. I think Heroku is just unhappy about something so it claims there are deployment errors? But they don't seem to come from anything in the change?

File: client/src/pages/WeeklyTable.js [hunk 1]
```diff
@@ -72,16 +79,20 @@ const getAvailableDesksForDay = (bookings, date, maxDesksForDay) => {
 
 const WeeklyTable = ({
 	bookings,
-	rowsCount,
 	refreshBooking,
 	maxDesksForDay,
 }) => {
+
+	const bookingsByRow = getBookingsByRow(bookings, week);
```
Comment (gregdyke) [inline client/src/pages/WeeklyTable.js]: this got merged into the wrong place?

**Review by gregdyke — COMMENTED** [review]
yeah, I'm mistaken :)

**Review by gregdyke — APPROVED** [review]

**Pull request merged**